enrolments/tests/serializers/test_session_list.py

```diff
@@ -0,0 +1,33 @@
+from django.test import TestCase
+
+from enrolments.models import Session
+from enrolments.serializers import (
+    ClassListSerializer,
+    SessionListSerializer,
+)
+from enrolments.tests.utils.utils import create_test_classes
+
+context = {"request": None}
+
+
+class SessionListSerializerTestCase(TestCase):
+    def setUp(self):
+        self.session = Session.objects.create(
+            name="Spring 2020",
+            fields=[1, 2, 3],
+        )
+        self.session_class = create_test_classes(self.session, 1)[0]
+        self.other_session_class = create_test_classes(self.session, 1)[0]
+
+    def test_session_detail_serializer(self):
```

Comment:
since this is testing session list serializer, I'm suggesting a rename
```suggestion
    def test_session_list_serializer(self):
```
also this test will need to change later as part of the refactor you wanted to do, but it's good for now imo

Reply:
oops good catch